requirements/requirements.cpu.txt

```diff
@@ -1,2 +1,2 @@
-onnxruntime<=1.15.1
+onnxruntime-openvino<=1.15.1
```

Comment:
Does onnxruntime-openvino work on arm based systems? I know it won't accelerate, but does it still function?

Reply:
the package doesnt even exist